go.mod

```diff
@@ -91,6 +91,7 @@ require (
 	k8s.io/apimachinery v0.0.0
 	k8s.io/client-go v0.0.0
 	k8s.io/helm v2.14.2+incompatible
+	k8s.io/klog v0.3.1
```

Comment:
Why are there changes to the go.mod?

Reply:
Not sure. I just ran `go build`. Sometimes the go.mod changes as a result. Maybe the module support in the newest go version does anything there? Removing the `indirect` is not a big deal but I don't know where `klog` comes from because it is not used anywhere in the files I edited. But it is used in `pkg/devspace/kubectl/transport/connection.go`, so it should have been in there beforehand. So, maybe go or the formatter in VS code fixes such things automatically now.